ui/components/app/permissions-connect-header/permissions-connect-header.component.js

```diff
@@ -71,6 +79,24 @@ export default class PermissionsConnectHeader extends Component {
           ) : null
           ///: END:ONLY_INCLUDE_IN
         }
+        {
+          ///: BEGIN:ONLY_INCLUDE_IN(flask)
+          snapVersion && (
+            <Typography
+              boxProps={{
+                margin: [2, 0],
+              }}
+              color={COLORS.TEXT_MUTED}
+              variant={TYPOGRAPHY.H7}
+              align={TEXT_ALIGN.CENTER}
+              tag="span"
+              className="version"
+            >
+              v{snapVersion}
```

Comment:
Do we want to consider using a localized `version` string here?  Is `v` meaningful in other languages?